Subscriber/CheckoutSubscriber.php

```diff
@@ -432,13 +441,18 @@ private function shouldRedirectToStep2(Shopware_Controllers_Frontend_Checkout $s
         $currency = Shopware()->Session()->sOrderVariables['sBasket']['sCurrencyName'];
         $value = Shopware()->Session()->sOrderVariables['sBasket']['AmountNumeric'];
 
+        if (empty((int) $value)) {
```

Comment:
Do we know what are the possible values for the `$value` ? In case it can be 0 or >0 then we can simply check `if (0 === $value)` instead of using empty. (I think in this case I would still avoid using empty and would just check after casting `$value` to int)

Reply:
It is usually a float variable (0.0), so `0 === (int)$value` or `0 ==$value` would all work just fine, it's just a matter of preference.

Reply:
I updated it to use the direct comparison